### PR TITLE
105 product detail vew

### PR DESCRIPTION
**Pull request by garrett-vangilder**
## Status

**READY/IN DEVELOPMENT**
## Documentation

[ ] I have fully documented features in this PR's code. Please put an `X` in the box to confirm.

[X] This PR does **NOT** require documentation.
## Description

A few sentences describing the overall goals of the pull request's commits.

This PR adds some small styling to the Product Detail View
## Resolves Issue Number
#105
## Deploy Notes

Notes regarding deployment the contained body of work.  These should note any
db migrations, etc.

``` sh
git pull --prune
git checkout 105-product-detail-view
dotnet run
```
## Steps to Test or Reproduce

Outline the steps to test or reproduce the PR here.
1. Go to a product view and look at that page. 
2. Sign in and go to a different product view for a product that you are not selling, then view a product you are selling.
3. Collect.
## Impacted Files in Application

List general components of the application that this PR will affect:
- Views/Products/Detail.cshtml

### CODE REVIEW

**Review by MattKraatz — APPROVED** [review]
LGTM

**Comment by matthamil** [comment]
LGTM :tada:

File: Views/Products/Detail.cshtml [hunk 1]
```diff
@@ -1,27 +1,46 @@
...
+   
         <div class="col-md-3"><a asp-controller="Cart" asp-action="AddToCart" asp-route-id="@Model.CurrentProduct.ProductId">Add to Cart</a></div>
     }
     else
     {
-        <a asp-controller="Products" asp-action="Delete" asp-route-id="@Model.CurrentProduct.ProductId"><span class="glyphicon glyphicon-remove" aria-hidden="true"></span></a>
-        <span>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span>
-        <a asp-controller="Products" asp-action="Edit" asp-route-id="@Model.CurrentProduct.ProductId"><span class="glyphicon glyphicon-edit" aria-hidden="true"></span></a>
+        <div class="col-md-3">        
+            <a asp-controller="Products" asp-action="Delete" asp-route-id="@Model.CurrentProduct.ProductId"><span class="glyphicon glyphicon-remove" aria-hidden="true"></span></a>
```
Comment (daynewright) [inline Views/Products/Detail.cshtml]: I think we should make the `Add To Cart` link into a button.  We can just add `class="btn btn-primary btn-xs"` to the `a` tag.

Comment (daynewright) [inline Views/Products/Detail.cshtml:38]: I think we should make the `Add To Cart` link into a button.  We can just add `class="btn btn-primary btn-xs"` to the `a` tag.

**Review by daynewright — CHANGES_REQUESTED** [review]
Can we change the `Add to Cart` to a button?

**Comment by Anulfo** [comment]
LGTM

File: Views/Products/Detail.cshtml (hunk 1 above)
Comment (garrett-vangilder) [reply Views/Products/Detail.cshtml:38]: Certainly

**Comment by garrett-vangilder** [comment]
@daynewright your button is here.

**Review by daynewright — APPROVED** [review]

**Pull request merged**